src/Cryptol/ModuleSystem/InstantiateModule.hs

```diff
@@ -217,6 +217,8 @@ instance Inst Expr where
         EProofApp e               -> EProofApp (go e)
         EWhere e ds               -> EWhere (go e) (inst env ds)
 
+        EPropGuards _guards       -> EPropGuards undefined -- TODO
+
```

Comment:
Don't worry about this---it doesn't exist in the new module system, and we'll have to implement it again there anyway.